corehq/apps/commtrack/templates/commtrack/manage/default_consumption.html

```diff
@@ -20,9 +20,11 @@
             <legend>{% trans 'Edit Default Monthly Consumption' %}</legend>
             {% crispy form %}
             <div class="form-actions">
-                <button type="submit" class="btn btn-primary">
-                    {% trans 'Update Default Consumption Info' %}
-                </button>
+                <div class="col-sm-12">
+                    <button type="submit" class="btn btn-primary">
+                        {% trans 'Update Default Consumption Info' %}
```

Comment:
since there isn't any logic changing the text for this button, is there a reason this doesn't live in the `crispy.Layout`?